Automattic/jetpack
# Jetpack Plugin: Fix dashboard backup link for Atomic sites

Fixes #26563

#### Changes proposed in this Pull Request:

* Adds an atomic check and uses the `calypso-backups` redirect instead of `my-jetpack-manage-backup` if the site is atomic

#### Other information:


#### Does this pull request change what data or activity we track or use?
No/



#### Testing instructions:





To test you'll need to utilize a WoA Dev site.
* Use Jetpack Beta or `jetpack rsync` after pulling and building the branch locally to install the plugin on your test site.
* Follow the instructions in https://github.com/Automattic/jetpack/issues/26563 and ensure the link now goes to the WordPress.com backups page for your site.

This checks specifically for WordPress.com on Atomic.  Ephemeral sites and Jurassic Ninja (and other test sites) should produce the `my-jetpack-manage-backup` redirect.  To test with these sites you'll need to manually add credentials before the link appears.
